src/hwlesson5/additional/task1/Main.java

```diff
@@ -0,0 +1,33 @@
+package hwlesson5.additional.task1;
+
+public class Main {
+    public static void main(String[] args) {
+
+        /**
+         * Найти в строке не только запятые, но и другие знаки препинания.
+         * Подсчитать общее их количество.
+         * */
+
+        String line = "sdf!#%^poiu}{lkj<>.,/asdqwert";
+        Main.punctuationMarksCounter(line);
```

Comment:
if we call method in this class, we can write only name of method without name of class